libraries/AP_HAL_ChibiOS/hwdef/BlitzH743Pro/hwdef.dat

```diff
@@ -0,0 +1,166 @@
+
+# hw definition file for processing by chibios_hwdef.py
+# for IFLIGHT_BLITZ_H7_PRO hardware.
+# thanks to betaflight for pin information
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# board ID for firmware load
+APJ_BOARD_ID 1063
+
+# crystal frequency, setup to use external oscillator
+OSCILLATOR_HZ 8000000
+
+FLASH_SIZE_KB 2048
+
+# bootloader takes first sector
+FLASH_RESERVE_START_KB 384
```

Comment:
That looks like more than the first sector....

Reply:
Yes includes the parameter pages